modules/50_damages/KWLike/bounds.gms

```diff
@@ -6,7 +6,10 @@
 *** |  Contact: remind@pik-potsdam.de
 *** SOF ./modules/50_damages/KWLike/bounds.gms
 
-loop(ttot$(ttot.val ge 2005),
+vm_damageProdFactor.fx(ttot,regi,in) = 1;
```

Comment:
is the out-commented line still needed? If yes, then maybe add a comment where it can be used for

Reply:
Took it out